### PR TITLE
Sort tasks

### PR DESCRIPTION
**Pull request by tyanhan**

### CODE REVIEW

File: src/main/java/seedu/address/model/schedule/Event.java [hunk 1]
```diff
@@ -99,6 +99,29 @@ public static boolean isValidEvent(Event event) {
...
+            return 1;
+        }
+
+        LocalTime eventTime = event.getTime();
+        if (time.isBefore(eventTime)) {
+            return -1;
+        } else if (time.isAfter(eventTime)) {
+            return 1;
+        } else {
+            return 0;
```
Comment (vishandi) [inline src/main/java/seedu/address/model/schedule/Event.java:121]: should we further compare the duration in this case?

Comment (tyanhan) [reply src/main/java/seedu/address/model/schedule/Event.java:121]: That would be possible. Should the longer event be first or the shorter event?

Comment (vishandi) [reply src/main/java/seedu/address/model/schedule/Event.java:121]: i think the shorter event should be first, as they finish first

**Comment by vishandi** [comment]
LGTM!

**Pull request merged**